config/routes.rb

```diff
@@ -1,11 +1,11 @@
 Rails.application.routes.draw do
-
+  # scope :module => 'foreman_gutterball', :path => '/gutterball' do
+  #   resources :content_reports
   scope :module => 'foreman_gutterball', :path => '/gutterball' do
-    resources :content_reports, :only => [:index, :show] do
-      member do
-        get :generate
+    scope :module => 'api' do
+      scope :module =>  'v2' do
```

Comment:
ah, oops. yes you're right. i think we can keep `/api` but foreman_gutterball doesn't have versioned routes so we can probably drop the `/v2`. if you need to move the controller file out of a v2 directory, that's fine.